src/Features/Blockcore.Features.NodeHost/UI/Shared/NavMenu.razor

```diff
@@ -24,11 +24,24 @@
             </NavLink>
         </li>
 
+
+
         @{
             var items = fullnode.NodeService<IEnumerable<INavigationItem>>();
             foreach (var item in items.OrderBy(i => i.NavOrder))
             {
-                if (item.IsVisible) {
+                if (item.IsVisible)
```

Comment:
instead of doing this better to add a mining nav item perhaps

Reply:
Done